src/content/details/faqs.mdx

```diff
@@ -119,10 +118,10 @@ We deploy extremely easily. We are completely infrastructure-agnostic. As long a
 
 **Can the inference engine run on Kubernetes? How do you containerize and take advantage of underlying infrastructure?**
 
-The DeepSparse Engine becomes a component of your model serving solution. As a result, it can
+DeepSparse becomes a component of your model serving solution. As a result, it can
 simply plug into an existing CI/CD deployment pipeline. How you deploy, where you deploy, and what you deploy on
-becomes abstracted to the DeepSparse Engine so you  can tailor your experiences. For example, you can run the
-DeepSparse Engine on a CPU VM environment, deployed via a Docker file and managed through a Kubernetes environment.
+becomes abstracted to DeepSparse so you can tailor your experiences. For example, you can run the
+DeepSparse on a CPU VM environment, deployed via a Docker file and managed through a Kubernetes environment.
```

Comment:
To confirm that this statement is true and makes sense @rsnm2 , it's applicable to ALL four components: Benchmark, Engine, Pipelines, and Server and not just the engine. Meaning, you should be able to substitute the phrase: "You can run all four components on a CPU VM environment, deployed via a Docker file and managed through a Kubernetes environment" and this content makes sense.

Reply:
It doesnt really make sense to use the Benchmark utilities inside K8S, but I think this is still fine to say. You might include a benchmark script in a CI/CD test or something